### DISCUSSION TITLE
Exporting to CC3 messes the character/rig up?

### DISCUSSION THREAD

**Discussion by Luckybonehead** [soupday/cc3_blender_tools#41] [Q&A]
Is this normal? When I import it to blender, everything's fine:
![Johnny Strong](https://user-images.githubusercontent.com/67667997/145095232-d508d560-d69d-4dbb-a483-19b070b1a113.PNG)

But when I export that from Blender to CC3, everything's messed up:

![Johnny Wrong](https://user-images.githubusercontent.com/67667997/145095314-61716def-f418-409e-9cd8-7937bafbcf0c.PNG)

I don't get any errors, so is this just an issue with the plugin right now? I'm hoping to import it to CC3 for a tutorial, but if I can't, I won't worry about it.

**Comment by soupday** [comment]
How exactly are you exporting to Blender, and how are you exporting it back?

Also is that an old character type (CC1 or CC2)?

I ask these because it doesn't look like a CC3+ character that's been imported by my Blender add-on (https://github.com/soupday/cc3_blender_tools). It looks like there's only diffuse textures, maybe some normals. Imported via the File > Import > FBX menu?

**Reply by Luckybonehead** [reply]
I'm exporting using CC3's "export" functions, and selecting "Blender" as the target tool preset. Then, I export from blender using the tool's "export to CC3" function.

Also, if I click on the mesh and rotate -90 on the X axis, it imports to CC3 just fine. Not sure why it doesn't look like I've imported it using the tools though; I've only been using CC3 for a couple of hours, so I'm not really sure what thing's should look like.

I think I see what you mean with the textures. My character gets into CC3, and I can use morph sliders on him, but he's untextured.

**Reply by soupday** [reply]
It just looks a bit flat around the eyes, but that could be the source character used. What is the base character you're using in CC3? The nearest I can find to this is the **CC3_Base Male** in the **Avatar/Base/Repack for CC3+** project templates.

I can't reproduce the rotation you're getting, and I tried resetting transforms and all sorts and they all came back in properly. But this looks like the kind of thing that used to happen a lot when importing/exporting by hand.

The character is supposed to come back fully textured... and standing up, without any intervention.

This is the full round trip process as a guide:

![1](https://user-images.githubusercontent.com/79094830/145179970-817fea9b-99a0-4af6-a212-e63f8dcf74c2.jpg)

![2](https://user-images.githubusercontent.com/79094830/145179987-2562da8e-55ce-4f05-95ef-472e21843743.jpg)

![3](https://user-images.githubusercontent.com/79094830/145179994-304bffa0-6d76-4302-a98a-5f794cd19e1f.jpg)

![4](https://user-images.githubusercontent.com/79094830/145180002-b5f2631c-a698-43ec-a7ea-ac0025ba24e6.jpg)

![5](https://user-images.githubusercontent.com/79094830/145180009-2f6044e1-c2a5-42a5-abe6-a6378a340a12.jpg)

Are you doing anything different to this?

**Reply by Luckybonehead** [reply]
Ah, I guess I was! I followed this tutorial and now everything imports and exports nicely. I guess I should have learned the software more before making this thread? Regardless, thanks for the help!
